lib/galaxy/managers/tags.py

```diff
@@ -198,7 +198,7 @@ def get_tags_str(self, tags):
             if tag.value is not None:
                 tag_str += ":" + tag.user_value
             tags_str_list.append(tag_str)
-        return ",".join(tags_str_list)
+        return ", ".join(tags_str_list)
```

Comment:
Hmm, I think I don't agree with this. There are many instances where we call `get_tags_str ` followed by `split(',')`. If this is for display purposes the downstream code should probably handle nicer formatting or get the tags as a list ? It's true that

Reply:
many places in the code should probably just receive the tags as a list or set, but that wouldn't be the minimal fix anymore and is something we should do in dev.

Reply:
@mvdbeek We shouldn't be composing a string and splitting it back apart multiple times in any event, this should be passed around as an array and formatted on display.  My goal here was to minimize potential side effects, which this does.  It's all due for an overhaul though.

Reply:
sounds good

Reply:
Maybe a new issue might be in order for refactoring this in dev? As @mvdbeek writes, there are other consumers that splits on comma and potentially other bugs. Agree this fix should be minimal.